packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -2,11 +2,13 @@ import { IIconProps } from '../../Icon';
 import { ISelectableOption } from '../../utilities/selectableOption/SelectableOption.Props';
 import { ISelectableDroppableTextProps } from '../../utilities/selectableOption/SelectableDroppableText.Props';
 
+export * from '../../utilities/selectableOption/SelectableOption.Props';

```

Comment:
why not follow what `Dropdown` does and have something like 
```
export interface IComboBoxOption extends ISelectableOption { }
```

and then `options?: IComboBoxOption`?